sources/move_library.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import sys
+import os
+from seaserv import seafile_api, ccnet_api
+
+seafile_local_emails = map(lambda user: user.email, ccnet_api.get_emailusers('DB', start=-1, limit=-1, is_active=None))
+active_user_emails = map(lambda user: user.email, ccnet_api.get_emailusers(source='LDAPImport', start=-1, limit=-1, is_active=True))
+ldap_user_emails = map(lambda user: user.email, ccnet_api.get_emailusers(source='LDAP', start=-1, limit=-1, is_active=None))
+non_active_user_emails = list(set(ldap_user_emails) - set(active_user_emails))
+
+seafile_domain = os.environ['SEAFILE_CURRENT_DOMAIN']
+source_user = sys.argv[1]
+source_user_email = source_user + '@' + seafile_domain
+assert (source_user_email in seafile_local_emails), 'The user must be in local seafile users list'
```

Comment:
Well it crash here because the old user is not in `seafile_local_emails`.

Reply:
With the old SSO behavior : Account created with SSO and not having LDAP matching account are in Seafile "DB".
So I will not move a account, if doesn't need to be.
Where is your old user ? Import Ldap  ?